value/pom.xml

```diff
@@ -139,33 +143,34 @@
 	    </goals>
 	    <configuration>
 	      <minimizeJar>true</minimizeJar>
-              <filters>
-                <filter>
-                   <artifact>org.apache.velocity:*</artifact>
-                   <includes>
-                       <include>**</include>
-                   </includes>
-                </filter>
-              </filters>
+          <artifactSet>
```

Comment:
Still looks off to me. `<artifactSet>` should be indented further than `<configuration>`.